mobify/progressive-web-scaffold
# Updates README, removes references to configure

**Linked PRs**: https://github.com/mobify/generator-progressive-web/pull/1, https://github.com/mobify/progressive-web-scaffold/pull/30

## Changes
- Updates README to be generic enough to also be used for generated projects
- Remove references to configure, which no longer is necessary

## How to test-drive this PR
- See linked PR above for generating a project. Also run this to ensure nothing has broken.
